src/QMCWaveFunctions/Fermion/SlaterDetBuilder.cpp

```diff
@@ -987,29 +953,29 @@ bool SlaterDetBuilder::readDetList(xmlNodePtr cur,
         RealType exctLvl, qc_ci = 0.0;
         OhmmsAttributeSet confAttrib;
         std::string tag, OccString;
-        #ifdef QMC_COMPLEX
-          RealType ci_real = 0.0, ci_imag = 0.0;
-          confAttrib.add(ci_real,"coeff_real");
-          confAttrib.add(ci_imag,"coeff_imag");
-        #else
-          RealType ci = 0.0;
-          confAttrib.add(ci, "coeff");
-        #endif
+#ifdef QMC_COMPLEX
```

Comment:
This and the next #ifdef can be removed through a fairly simple templated function refactor.  See https://github.com/anbenali/qmcpack/pull/12

Reply:
This is updated. Thanks for the fix!